cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -209,21 +216,47 @@ def test_vm_reconfig_add_remove_hw_hot_vmware(change_type):
         casecomponent: Infra
         tags: reconfigure
         testSteps:
-            1. Change number of CPU sockets and amount of memory while VM is runnng.
+            1. Add and remove the disk while VM is running
             2. Go to Compute -> infrastructure -> Virtual Machines -> Select Vm
             3. Go to VM reconfiguration
-            4.Change number of CPU sockets and amount of memory, save and submit
+            4. Click on Add Disk -> select disk_type and disk_mode , save and submit
             5. Check the count in VM details page
+            6. Remove the disk and Check the count in VM details page
     """
-    pass
+    orig_config = small_vm.configuration.copy()
+    new_config = orig_config.copy()
+    new_config.add_disk(
+        size=500, size_unit='MB', type=disk_type, mode=disk_mode)
+
+    add_disk_request = small_vm.reconfigure(new_config)
+    # Add disk request verification
+    wait_for(add_disk_request.is_succeeded, timeout=360, delay=45,
+             message="confirm that disk was added")
+    # Add disk UI verification
+    wait_for(
+        lambda: small_vm.configuration.num_disks == new_config.num_disks, timeout=360, delay=45,
+        fail_func=small_vm.refresh_relationships,
+        message="confirm that disk was added")
+    msg = "Disk wasn't added to VM config"
+    assert small_vm.configuration.num_disks == new_config.num_disks, msg
```

Comment:
Question: Why do you include `msg` here?

Reply:
@john-dupuy msg is just for printing the message if assertion will fail.
https://docs.pytest.org/en/3.0.3/assert.html?highlight=re#asserting-with-the-assert-statement

Reply:
Thanks, I wasn't aware of that!